argon2/src/lib.rs

```diff
@@ -100,12 +100,16 @@ use password_hash::{Decimal, Ident, ParamsString, Salt};
 /// Maximum password length in bytes.
 pub const MAX_PWD_LEN: usize = 0xFFFFFFFF;
 
-/// Minimum and maximum salt length in bytes.
+/// Minimum salt length in bytes.
 pub const MIN_SALT_LEN: usize = 8;
 
 /// Maximum salt length in bytes.
 pub const MAX_SALT_LEN: usize = 0xFFFFFFFF;
 
+/// Recommended salt length for password hashing in bytes.
+#[cfg(all(feature = "password-hash"))]
```

Comment:
```suggestion
#[cfg(feature = "password-hash")]
#[cfg_attr(docsrs, doc(cfg(feature = "password-hash")))]
```

Reply:
Applied in d298106586d3cf20b0e24372e0fcb600f15c5987.